src/sass/components/_contacts.scss

```diff
@@ -4,12 +4,13 @@
     background-color: rgba(255, 245, 246, 1);
     padding-top: 50px;
     padding-bottom: 50px;
+
 }
 
 .contacts__list {
     list-style: none;
     padding: 0;
-    margin: 0;
+    transform: translateY(-166px);
```

Comment:
translate robi masakrę w tym momencie w sąsiednich sekcjach. Czy uzgodniłeś to z Mikołajem Marcinkowskim (footer) i Moniką Cihocką? (customer reviews)

Reply:
Razem z gradientem przesunięcie translateY(-166px) nie jest potrzebne, wyciągam z repo.